napalm_logs/config/opengear/SIGNAL.yml

```diff
@@ -0,0 +1,14 @@
+#<30>ztp-tankstelle_1-ts port02 2018-Nov- 9 15:42:06.972 SIGNAL: CTS => 0
+
+messages:
+  - error: SIGNAL
```

Comment:
In what specific context would this notification be used, for example?

Reply:
The opengear console server detects serial signal changes. This is the message for it. Currently maybe there is no use for it but I included it because I had it already.